packages/replay-internal/test/integration/eventProcessors.test.ts

```diff
@@ -48,8 +50,7 @@ describe('Integration | eventProcessors', () => {
     const TEST_EVENT = getTestEventIncremental({ timestamp: BASE_TIMESTAMP });
 
     mockRecord._emitter(TEST_EVENT);
-    jest.runAllTimers();
-    jest.advanceTimersByTime(1);
+    vi.runAllTimers();
```

Comment:
I think I'll do a separate PR to update how we use fake timers to remove nextTick calls and change runAllTimers to advancedToNextTimerAsync or runAllTimersAsync